### DISCUSSION TITLE
Inertia React + Mantine UI

### DISCUSSION THREAD

**Discussion by rahmat-dev** [mantinedev/mantine#4709] [Q&A]
Can I use mantine with my laravel project that has been installed react inertia?

**Comment by rtivital** [comment]
Yes, if the stack supports React, then you can use Mantine components with it.

**Reply by rahmat-dev** [reply]
but I'm having a problem with the button component not showing, but when I hover the component is showing

**Reply by rtivital** [reply]
See https://tailwindcss.com/docs/preflight#disabling-preflight
